virttest/cpu.py

```diff
@@ -1050,6 +1050,42 @@ def _make_up_pattern(flags):
     return cpu_support_model
 
 
+def parse_qemu_cpu_models_modern(help_text):
```

Comment:
hi @luckyh , do we expect the users to use this function in future to get the cpu model list? This is a public function which can cover both the old and new version of qemu

Reply:
@zhencliu I believe the old one (`extract_qemu_cpu_models`) would be still the best choice for users in a long time, since it automatically invoked the new one within its implementation therefore no check will be required when users intend to call the functionality.